### DISCUSSION TITLE
Bandwidth usage exposed in Vercel API

### DISCUSSION THREAD

**Discussion by bmstefanski** [vercel/community#933] [Idea]
Hi Vercel team, thanks for the great platform, libraries, and legendary Next.js 😎 

I just skimmed through [Vercel API Reference](https://vercel.com/docs/rest-api) and it seems like you can't get info about bandwidth usage from it. There's a weak workaround, I could fetch how much I pay for the bandwidth (`invoiceItems.bandwidth`) and calculate based on that, but it wouldn't work well for Enterprise teams since the price isn't fixed.

It would be beneficial if we had such a field exposed in the API, especially for the ones who host user-generated websites on Vercel and want to limit their bandwidth.

Thanks in advance for the response

**Comment by nitriques** [comment]
Yeah we would also need to be able to monitor the bandwidth usage per project (and all other metrics we currently find in the dashboard).

Why is this not exposed ?
